### PR TITLE
Update messaging to mention MicroMasters

### PR DESCRIPTION
**Pull request by andy-armstrong**
https://openedx.atlassian.net/browse/TNL-6113

This is a one line text change, along with corresponding test fixes.

@catong @dianakhuang, please review.

### CODE REVIEW

File: edx_proctoring/templates/proctored_exam/entrance.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
   </h3>
   <p>
     {% blocktrans %}
-      To be eligible to earn credit for this course, you must take and pass the proctoring review for this exam.
+      To be eligible for course credit or for a MicroMasters credential, you must pass the proctoring review for this exam.
```
Comment (catong) [inline edx_proctoring/templates/proctored_exam/entrance.html]: Should we make "MicroMasters" a variable?

**Comment by catong** [comment]
👍 with my question about hardcoding "MicroMasters".

File: edx_proctoring/templates/proctored_exam/entrance.html (hunk 1 above)
Comment (andy-armstrong) [inline edx_proctoring/templates/proctored_exam/entrance.html]: I considered that, but then saw that it is hard-coded in other strings in edX platform. It is an interesting point that this message is very edX-specific now, which isn't ideal for an open source library. @dianakhuang do you have any recommendations?

**Comment by dianakhuang** [comment]
👍  from me as well.

**Pull request merged**